packages/swing-store/docs/kvstore.md

```diff
@@ -0,0 +1,33 @@
+# KVStore
+
+The `kernelStorage.kvStore` sub-store manages a table of arbitrary key-value (string-to-string) pairs. It provides the usual get/set/has/delete APIs, plus a `getNextKey` call to support lexicographic iteration.
+
+There are three separate sections of the namespace. The normal one is the "consensus" section.  Each value written here will be given an export-data row, and incorporated into the "crankhash" (described below).
```

Comment:
yeah, it's exactly "anything that doesn't start with `local.` or `host.`, because "consensus" came first, and because there are maybe three keys total in the local/host section

I'll update the text.